bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/servlet/SkeletonServlet.scala

```diff
@@ -104,6 +104,10 @@ final class SkeletonServlet(odb: IDBDatabaseService, templateFactory: TemplateFa
         a => a.asInstanceOf[FileItem]
       })
 
+  /** If `test` is set then the PDF file is optional and ignored if present. */
+  private def test(items: List[FileItem]): Boolean =
+    items.exists(_.getFieldName == "test")
```

Comment:
I added it to the servlet so you can skip the PDF and attachment processing. Thanks for pointing this out; I had forgotten about it and need to write a test.